src/Command/DeleteAllSchemasCommand.php

```diff
@@ -29,8 +36,16 @@ public function execute(InputInterface $input, OutputInterface $output): int
     {
         $schemas = $this->schemaRegistryApi->getSubjects();
 
+        $hardDelete = (bool) $input->getOption('hard');
+
         foreach ($schemas as $schemaName) {
             $this->schemaRegistryApi->deleteSubject($schemaName);
+
+            if ($hardDelete) {
+                $this->schemaRegistryApi->deleteSubject(
```

Comment:
Do we still need to call `deleteSubject` from above in this case or we can just decide depending on a flag which one should be called?

Reply:
from: https://docs.confluent.io/platform/current/schema-registry/schema-deletion-guidelines.html
> To accomplish a hard delete of a schema (all versions or a specific version), use this two-step process. You must first soft delete the schema, then hard-delete it.

So we need both calls to actually execute the process of "hard delete of schema".

Reply:
Cool, thanks for the info 👍